docs/pages/careers/react-engineer-x-charts.md

```diff
@@ -0,0 +1,139 @@
+# React Engineer - xCharts
+
+<p class="description">You will help form the xCharts team, build ambitious and complex new features, work on strategic problems, and help grow adoption.</p>
+
+## Details of the Role
+
+- **Location**: Remote (preference for UTC-6 to UTC+5).
```

Comment:
Nitpick: Maybe express a strong preference or even a requirement in case of the current X team composition/location? 🤔 
```suggestion
- **Location**: Remote (strong preference for UTC-6 to UTC+5).
```